src/Microsoft.AspNetCore.Mvc.RazorPages/breakingchanges.netcore.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "TypeId": "public class Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageResultExecutor : Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor",
+    "MemberId": "public .ctor(Microsoft.AspNetCore.Mvc.Internal.IHttpResponseStreamWriterFactory writerFactory, Microsoft.AspNetCore.Mvc.ViewEngines.ICompositeViewEngine compositeViewEngine, Microsoft.AspNetCore.Mvc.Razor.IRazorViewEngine razorViewEngine, Microsoft.AspNetCore.Mvc.Razor.IRazorPageActivator razorPageActivator, System.Diagnostics.DiagnosticSource diagnosticSource, System.Text.Encodings.Web.HtmlEncoder htmlEncoder)",
+    "Kind": "Removal"
+  }
+]
```

Comment:
@Eilon the new pseudo-breaking change records here and in Microsoft.AspNetCore.Mvc.ViewFeatures are like the ones above (in Microsoft.AspNetCore.Mvc.Core) but for `IHttpResponseStreamWriterFactory`'s move. Do you approve these as well?

Reply:
Yeah this is fine because it uses `.Internal`, so not breaking.